app/main.py

```diff
@@ -1,3 +1,65 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __str__(self) -> None:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> None:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: int | float) -> str:
+        if isinstance(other, Distance):
+            return Distance(self.km + other.km)
+        elif isinstance(other(int, float)):
+            return Distance(self.km + other.km)
```

Comment:
In case `other` is `int` or `float`, it does not have the attribute `km`. Add the value directly.